### PR TITLE
Contact form pull request

### PR DESCRIPTION
**Pull request by kirube1992**
Add forms
Add style
use new background

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,5 +160,19 @@ <h1 class="item1 h1">I’m Kirubel. <br>Glad to see<br> you!</h1>
...
     </section>
+    <section class="contact">
+      <h2 class="me">Contact me</h2>
+      <p class="mep">If you have an application you are interested in developing, a feature
+         that you need built or a project that needs coding.
+        I’d love to help with it!</p>
+    <form action="https://formspree.io/f/xyyanzvl" method="post">
+      <input name="name" type="text" placeholder="Full Name" maxlength="30" required>
+      <input name="email" type="email" placeholder="Email address" required>
+      <textarea name="text-box" maxlength="500" placeholder="Write your comment here"></textarea>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure that your `textarea` field has the `required` attribute to activate the validation on submission. :+1:

File: style.css [hunk 2]
```diff
@@ -253,3 +253,81 @@ section ul {
...
+
+textarea {
+  padding: 8px 5px;
+  color: #cfd8dc;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 100px;
+  margin: 5px 0;
+  font-family: "Poppins", sans-serif;
+}
```
Comment (Owusu-Desmond) [inline style.css:311]: - [ OPTIONAL ] Why dont you make the text in the textarea easily seen to improve the accessibility of your project :pray: 
  
  image

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @kirube1992 and @PabloEGonz  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @team 

Your project is complete! Great job implementing the changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- form submits well
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**